httpx_auth/_oauth2/implicit.py

```diff
@@ -104,17 +101,16 @@ def __init__(self, authorization_url: str, **kwargs):
             self.redirect_uri_port,
         )
 
-    def auth_flow(
-        self, request: httpx.Request
-    ) -> Generator[httpx.Request, httpx.Response, None]:
-        token = OAuth2.token_cache.get_token(
-            self.state,
-            early_expiry=self.early_expiry,
-            on_missing_token=authentication_responses_server.request_new_grant,
-            grant_details=self.grant_details,
```

Comment:
It looks like this change removed the need for on_missing_token kwargs to be provided right?

Reply:
This is a nice change, could you just add a TODO on the get_token cache method to change the signature to remove this as its now unused? I'll do it myself once we have the whole async stuff ready :)

Reply:
Small enough change, and closely related...